accounts-management/manager_test.go

```diff
@@ -112,7 +114,17 @@ func TestVerifyAccountPassword(t *testing.T) {
 		require.NoError(t, err)
 
 		if testCase.importToLocation {
-			err = os.Link(filename, filepath.Join(keystore.KeystorePath(), filepath.Base(filename)))
+			// Copy file instead of hard link to support cross-filesystem operations
```

Comment:
Hmm, linking didn't work on your platform? Which OS is that?
I prefer the linking as it's much simpler. But if you say it doesn't work on some OS, then we should extract this code into some function. We can do it in a separate PR.